Volha_Ivanova/Module_1_Java/Java_Fundamentals/HomeWork1/src/com/epam/cdp/volha_ivanova/task3/Calculator.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.cdp.volha_ivanova.task3;
+
+public class Calculator {
+	
+	private int value1;
+	private int value2;
+	
+	  public Calculator(int value1, int value2) {
+	    this.value1 = value1;
+	    this.value2 = value2;	    	    
+	  }
+	  
+	  public int sum() { //Method for Summation
+	    int result = 0;
+	    result = value1 + value2;
+	    return result;
+	  }
+
+	  public int sub() { //Method for Subtraction
+		 int result = 0;
+		 result = value1 - value2;
+		 return result;
+	  }
+	  
+	  public int mult() { //Method for Multiplication
+		int result = 0;
+		result = value1 * value2;
+		return result;
+	  }
+	  
+	  public int div() { //Method for Division
```

Comment:
check or catch division by zero in main application. Your app should not crash due to incorrect data
and again
just
``return value1/value2``